spec/extra/sanity.json

```diff
@@ -82,9 +84,9 @@
         "valid": true
       },
       {
-        "description": "ignores non-objects (empty array version) - AGAINST SPEC",
+        "description": "ignores non-objects (empty array version) - AGAINST SPEC ???",
         "data": [],
-        "valid": false
+        "valid": true
```

Comment:
Same here. I do understand that these tests were copied from the original forked library and a lot of changes were made.

Reply:
same answer as above... https://github.com/Tieske/lua-resty-ljsonschema/pull/10#discussion_r513453188